gotocompany/compass/v1beta1/service.proto

```diff
@@ -1146,6 +1178,58 @@ message DeleteAssetRequest {
 
 message DeleteAssetResponse {}
 
+message DeleteAssetsRequest {
+  string q = 1 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by specific query"
+    }
+  ];
+  string q_fields = 2 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by multiple query fields"
+    }
+  ];
```

Comment:
i don't think we need this for now

Reply:
instead, we need filter like this
https://github.com/goto/proton/blob/64dd189a923a3474f3ece43542313ee264430a2b/gotocompany/compass/v1beta1/service.proto#L837-L839

Reply:
Okay, I will remove q and q_fields ya, and It is similar to `map<string, string> data = 5`, but I will replace as your mention code 👍